### PR TITLE
Remove "id" field

### LINKED ISSUE #5: Remove "id" field
The registry concerns itself mostly with the network field and the id field was meant to be used as DID to resolve a document part of this registry. However it makes more sense to remove it since DID Documents are meant to be authenticated by the owner but in this scenario we have a community managed registry which is not owned by anyone but maintained by everyone

### PR DESCRIPTION
**Pull request by pedrouid**
Fix #5

### CODE REVIEW

File: evm.json [hunk 1]
```diff
@@ -1006,5 +897,66 @@
...
+        "exponent": 18
+      }
+    ],
+    "rpc": [
+      "https://pyrus2.ubiqscan.io"
+    ],
+    "custom": {
+      "chainId": 8,
+      "networkId": 1
+    }
```
Comment (webmaster128) [inline evm.json]: Why such a big diff? For some reason the order of the chains changed.

Comment (pedrouid) [reply evm.json]: Arghh I didn't notice that, but on `scripts/shared/index.js` you can see on L184 that `saveList` orders the output json by `id` which was removed. I will change it order by `network`

Comment (pedrouid) [reply evm.json]: Plus I forgot to update the standard spec

Comment (pedrouid) [reply evm.json]: Cool, ordering by `network` reduced the diff

Comment (pedrouid) [reply evm.json]: My IDE also auto-formatted the spec md 😅 I just reverted it

File: scripts/shared/index.js [hunk 2]
```diff
@@ -179,30 +179,9 @@ async function verifyJson (json) {
...
-    )
-  }
-  const did = `did:${method}:${json.network}`
-  return did
-}
-
 async function saveList (array, title, log = true) {
   const filePath = path.join(ROOT_DIRECTORY, `${title}.json`)
-  const json = sortBy(array, ['id'])
+  const json = sortBy(array, ['network'])
```
Comment (webmaster128) [inline scripts/shared/index.js]: Note: with this change in first, #16 needs to be updated

**Review by webmaster128 — APPROVED** [review]
Nice!

**Comment by webmaster128** [comment]
Feel free to merge when you're happy with this

**Review by Pazibad — COMMENTED** [review]
constants/extraRpcs.js

**Pull request merged**